packages/code-du-travail-frontend/pages/widgets/modeles-de-courriers/[id].tsx

```diff
@@ -20,7 +20,7 @@ function Widgets(props: LetterModelProps): JSX.Element {
         overrideCanonical={SITE_URL + "/modeles-de-courriers"}
       />
       <StyledHeader>
-        <StyledTitle small stripe="left">
+        <StyledTitle stripe="left" variant="secondary">
```

Comment:
j'ai fait ça pour enlever la property small sur le component PageTitle (qui n'était utilisé qu'ici) mais qui affichait le warning suivant sur toutes nos pages: 
```
Warning: Failed prop type: PageTitle: prop type `small` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.This often happens because of typos such as `PropTypes.function` instead of `PropTypes.func`.
PageTitle@webpack-internal:///../react-ui/lib/Titles/PageTitle/index.js:161:14
```